### DISCUSSION TITLE
Trojan:Script.Phonzy.C!ml

### DISCUSSION THREAD

**Discussion by howefield** [ventoy/Ventoy#1206] [General]
Windows defender picking up trojan in Windows download of ventoy-1.0.58-windows.zip

**Comment by ventoy** [comment]
It's false positive.
There is such false positive when I update Ventoy2Disk.exe, but after some time it will be added to the whitelist.
This happened too many times.

#1204
#1203
#1194
#660
#468
#371
#31

**Comment by howefield** [comment]
Thank you, and apologies for not finding the previous times.
